src/settings-ui/Settings.UI/SettingsXAML/App.xaml.cs

```diff
@@ -66,7 +66,8 @@ private enum Arguments
 
         public bool ShowScoobe { get; set; }
 
-        public Type StartupPage { get; set; } = typeof(Views.DashboardPage);
+        // TODO(stefan): REVERT THIS TO DASHBOARD PAGE!!!! SPELCHHHHEEK FAIL
```

Comment:
This is a workaround to be able to start Settings app. Because Dashboard uses staggered layout and there's a crash when loading DashboardPage. We discussed this, it's caused by Toolkit nuget package update